crates/forge/src/result.rs

```diff
@@ -472,42 +529,50 @@ impl TestKindReport {
     }
 }
 
-/// Various types of tests
+/// Various types of test environments
 #[derive(Clone, Debug, Serialize, Deserialize)]
-pub enum TestKind {
-    /// A standard test that consists of calling the defined solidity function
-    ///
-    /// Holds the consumed gas
-    Standard(u64),
-    /// A solidity fuzz test, that stores all test cases
-    Fuzz {
-        /// we keep this for the debugger
-        first_case: FuzzCase,
-        runs: usize,
-        mean_gas: u64,
-        median_gas: u64,
+pub enum TestEnvironment {
+    /// A standard test environment
+    Standard,
+    /// A forked test environment
+    Fork {
+        /// The block number at which the test was executed
+        block_number: u64,
```

Comment:
the block number is used for https://github.com/foundry-rs/foundry/issues/7574

Reply:
Since been updated, I've added a test case for tracking multiple forks. For now we collect the block heights of the multiple forks but only refer to the last one (the failure case) when we decorate the failure case with the block height